src/main/java/com/sigma/cinematicketpro/exception/ApiErrorResponse.java

```diff
@@ -2,15 +2,13 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import lombok.AllArgsConstructor;
-import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Getter
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @NoArgsConstructor
 @AllArgsConstructor
-@EqualsAndHashCode
 public class ApiErrorResponse {
     private int code;
```

Comment:
this can be a special type http response type ;) not int

Reply:
I guess you are referring to HttpStatus enum. But the spring will serialize it as a string, and I want the user to receive response code as an integer. Similar approach is used by Google https://cloud.google.com/apis/design/errors